composer.json

```diff
@@ -9,6 +9,7 @@
         "php": "^7.2"
     },
     "require-dev": {
```

Comment:
Should this stay in the required section?

Reply:
Yeah, absolutely, my bad